File: Java code/src/main/java/VIA/VIATalks/Database/data/University.java
package VIA.VIATalks.Database.data;

import java.io.Serializable;

public class University implements Serializable {
    private int id;
    private String name;
    private String country;

    //Constructor
    public University(int id, String name, String country) {
        this.id = id;
        this.name = name;
        this.country = country;
    }

    //Getters and Setters

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }
}

File: C# code/VIATalks/Shared/University.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Shared
{
    [Serializable]
    class University
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Country { get; set; }

        public University(int id, string name, string country)
        {
            Id = id;
            Name = name;
            Country = country;
        }
    }
}
